### ISSUE TITLE
Text overlaps image in "rounded buttons" in generated dashboard

### ISSUE THREAD

**Issue by mvdwetering** [home-assistant/frontend#19671]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

In the autogenerated dashboard there are rounded buttons (don't know official name) at the bottom of room cards with scenes (for me it is hue scenes, but it applies to all entities with `entity_picture` assigned). The text overlaps with the image.

![image](https://github.com/home-assistant/frontend/assets/732514/cfcf35e5-759e-4097-8ce5-66d039b166fa)

### Describe the behavior you expected

No overlap with the image.

### Steps to reproduce the issue

1. Have a room with some (hue?) scenes and other entities.
2. Go to the autogenerated dashboard

### What version of Home Assistant Core has the issue?

2024.1.6

### In which browser are you experiencing the issue with?

Versie 120.0.6099.225 (Officiële build) (64-bits)

### Which operating system are you using to run this browser?

Windows 11

### State of relevant entities

```txt
group_name: Toilet
group_type: room
name: Gedimd
speed: 0.5
brightness: 30.04
is_dynamic: false
friendly_name: Toilet Gedimd
entity_picture: /hue_scene_images/gedimd.webp
```

### Problem-relevant frontend configuration

```yaml
Don't know, it is on the autogenerated dashboard
```

### Additional information

I am pretty sure the text used to be a bit more to the right and not overlap, but not sure when it changed as I don't visit the automatic dashboard that often.

Edit:
I actually learned that these buttons are footer buttons and you can get them for any entity, it is documented here https://www.home-assistant.io/dashboards/header-footer/

This is an example that should work for any installation with an entities card

```
type: entities
entities:
  - sun.sun
footer:
  type: buttons
  entities:
    - entity: sun.sun
      name: "Example name"
      image: "https://placehold.co/100x100/000000/FFF"
```

![image](https://github.com/home-assistant/frontend/assets/732514/7bc9fd10-752f-4748-b3f6-e5e7379f4fa8)

**Comment by mvdwetering** [comment]
I bumped into an old image and that had correct spacing. That image was from September 10th 2023, not sure if I was running 2023.9.x already then, could still have been 2023.8.x. So it seems to have changed between 2023.8.x and 2024.1

Old spacing:
![image](https://github.com/home-assistant/frontend/assets/732514/186bb333-8689-46fc-b02d-fa48f0826f34)

Current spacing (same yaml code):
![image](https://github.com/home-assistant/frontend/assets/732514/33fe0d5f-b6f0-4a72-973c-6e30f47543c2)

**Comment by mvdwetering** [comment]
Was fixed in 2024.4.3 by #20492
